functions/model/librarian/tark/deleteFilesInOrg.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable max-len */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable object-curly-spacing */
+/* eslint-disable no-unused-vars */
+
+const { updateFileToTask } = require("../lib");
+const { updateFileToOrg } = require("../lib")
```

Comment:
const { updateFileToOrg } = require("../lib"). This statement is written twice. Remove one.